services/brig/Makefile

```diff
@@ -89,6 +89,10 @@ integration-list: install
 integration-%: db-migrate index-reset
 	LOG_LEVEL=Info keiretsu --run '$(EXE_IT) --pattern="$*"' --env $(KEIRETSU_ENV)
 
+test-fast:
+	stack install --fast --test --local-bin-path=dist
+	cp "$(shell stack path --dist-dir)/build/$(NAME)-integration/$(NAME)-integration" dist/
```

Comment:
I don't think copying is needed anymore (integration test is defined as executable now in .cabal file). Also `test:` should be sufficient as a target.

Reply:
Yup, agreed